CHANGELOG.md

```diff
@@ -3,9 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
-## [?]
+## [1.1.10] - 2023-10-03
 ### Fixed
 * Incorrect BatchUpdateException.getUpdateCounts() on failed batch statement execution ([Issue #450](https://github.com/awslabs/aws-mysql-jdbc/issues/450)).
+* Added missing documentation regarding the `enableFailoverStrictReader` parameter ([Issue #456](https://github.com/awslabs/aws-mysql-jdbc/issues/456).
```

Comment:
Nit: No need to mention that the documentation was missing.

Reply:
What about "improved documentation around..."